frontend_app_kanban/src/css/index.styl

```diff
@@ -13,12 +14,12 @@ cardWidth = 280px
       display flex
       flex-direction column
       // INFO - GB - 2021-12-20 - 28px is the wsContentGeneric__content__left__top height
-      height "calc(100% - %s)" % 28px
+      height "calc(100% - %s)" % (headerHeight + 28px)
       &.fullscreen
         position fixed
         width 100%
         height 100%
-        z-index 1
+        z-index 2
```

Comment:
same comment about z-index

z-index of 1 doesn't require extra information because it is often used to put an element over another one without z-index.
But 2 means you put it above something else that has a z-index

Reply:
Done